### PR TITLE
Reindent header.hbs

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

**Comment by XhmikosR** [comment]
Note that the indentation error wasn't caught by htmllint. Just pointing it out.

~~As for the end tags, they are not needed.~~

File: layouts/partials/header.hbs [hunk 1]
```diff
@@ -23,11 +23,11 @@
       </ul>
     </nav>
 
-  <svg id="lang-picker-toggler" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
-    <rect id="lang-rect" width="512" height="512" x="0" y="0" fill="none" pointer-events="fill"/>
```
Comment (osk2) [inline layouts/partials/header.hbs:27]: I don't think slash of self-close tag can be omitted. 🤔

Comment (XhmikosR) [reply layouts/partials/header.hbs:27]: Of course it can :)

Comment (XhmikosR) [reply layouts/partials/header.hbs:27]: Actually, you are right. Let me try something and I'll sort this out.

Comment (XhmikosR) [reply layouts/partials/header.hbs:27]: Ok fixed. Not sure how I got confused here :/

Comment (osk2) [reply layouts/partials/header.hbs:27]: 👍

**Review by osk2 — APPROVED** [review]
LGTM

**Pull request merged**